pkg/cluster/graph/manager.go

```diff
@@ -4,12 +4,11 @@ package graph
 // Licensed under the Apache License 2.0.
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
 	"io"
 
-	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/storage/armstorage"
```

Comment:
The import is inconsistent.  `hack/validate-imports/allowed-import-names.yaml` has us stating this will be `sdkstorage`.  If you dont' mind picking one `armstorage` or the other `sdkstorage`.

Reply:
used `storagesdk` for the less frequent import now.